src/sentry/runner/commands/configoptions.py

```diff
@@ -0,0 +1,150 @@
+import click
+
+from sentry.runner.decorators import configuration
+
+
+@click.group()
+def configoptions():
+    "Manages Sentry options."
+
+
+@configoptions.command()
+@click.option("--key", "-k", required=True, help="The key of the option to fetch.")
+@configuration
+def fetch(key):
```

Comment:
While not mandatory, it would be a good idea to add type hints. That makes your code more robust and bugs easier to spot. We run the type checker in CI

Reply:
Good call. We'll definitely be doing more code hygiene as this project develops.